livekit-android-sdk/src/test/java/io/livekit/android/room/SignalClientTest.kt

```diff
@@ -229,6 +229,18 @@ class SignalClientTest {
             build()
         }
 
+        val STREAM_STATE_UPDATE = with(LivekitRtc.SignalResponse.newBuilder()) {
+            streamStateUpdate = with(LivekitRtc.StreamStateUpdate.newBuilder()) {
+                addStreamStates(with(LivekitRtc.StreamStateInfo.newBuilder()) {
+                    participantSid = TestData.REMOTE_PARTICIPANT.sid
+                    trackSid = TestData.REMOTE_AUDIO_TRACK.sid
```

Comment:
This is test. So, not a big deal. But, just wanted to mention `StreamStateUpdate` is only for video tracks. Audio tracks are not paused/resumed. They are always forwarded

Reply:
Good to know!